CHANGELOG.md

```diff
@@ -3,11 +3,12 @@
 ### Bug fixes
 
 * Fixed log levels in JNI layer (#4204).
+* Fixed a bug in encryption (#4128).
```

Comment:
Should we write more? Like it causing apps in rare cases wh n using many threads?